src/system.rs

```diff
@@ -143,7 +143,7 @@ fn parse_proc_cpuinfo() -> Result<Model> {
         // Older revisions are 4 characters long, or 8 if they've been over-volted
         match &revision[revision.len() - 4..] {
             "0007" | "0008" | "0009" | "0015" => Model::RaspberryPiA,
-            "Beta" | "0002" | "0003" => Model::RaspberryPiBRev1,
+            "beta" | "0002" | "0003" => Model::RaspberryPiBRev1,
```

Comment:
Can you provide any details for this change? I don't have access to a beta model of the first Pi myself, but I based the early 4-character revision codes on the list available at https://elinux.org/RPi_HardwareHistory which specifies `Beta` with a capital B. (The revision codes at https://www.raspberrypi.org/documentation/hardware/raspberrypi/revision-codes/README.md don't list the beta model)

Reply:
Above we are calling

```rust
revision = String::from(&line[11..]).to_lowercase();
```

so it will never use a capital “B”.